src/patient-registration/patient-registration.resource.tsx

```diff
@@ -12,6 +12,17 @@ export function savePatient(abortController: AbortController, patient: Patient)
   });
 }
 
+export function saveRelationships(abortController: AbortController, relationships: Object) {
```

Comment:
Could we merge this with the `savePatient()` to remove some code duplication?

Reply:
This would imply introducing state management in our resource class since the relationships are saved one by one...Quick thought would be to have a utility folder responsible for such scenarios such that we can keep our component clean with less crowding with such functions....Though not sure how easy it would be

Reply:
We can look into it further, currently I think it is more of a small tech debt rather than something that is required for this PR.